src/app/ipv/middleware.js

```diff
@@ -409,7 +409,7 @@ module.exports = {
   validateFeatureSet: async (req, res, next) => {
     try {
       const featureSet = req.query.featureSet;
-      const isValidFeatureSet = /^\w{1,32}$/.test(featureSet);
+      const isValidFeatureSet = /^(?!,)((,)?(\w{1,32})\b)*$/.test(featureSet);
```

Comment:
It might be more readable without the negative lookahead and word boundary, but this does work.

e.g. `/^\w{1,32}(,\w{1,32})*$/`

Reply:
Done...aded few more tests.